### PR TITLE
docs(vscode.mdx): translate `vscode.mdx` to `pt-br`

### PR DESCRIPTION
**Pull request by ImBIOS**
## Summary

Part of:
- #1065 

## Test Plan

_null (it's a joke for any rust project. Rust have no null, 🕵️  right?)_

### CODE REVIEW

File: website/src/content/docs/pt-br/reference/vscode.mdx [hunk 1]
```diff
@@ -1,76 +1,78 @@
...
 
-## Installation
+## Instalação
 
-You can install the code extension by heading to the extension's [Visual Studio Code Market Place page](https://marketplace.visualstudio.com/items?itemName=biomejs.biome) or from within VS Code by either:
+Você pode instalar a extensão do código acessando a [página do Visual Studio Code Market Place da extensão](https://marketplace.visualstudio.com/items?itemName=biomejs.biome) ou dentro do VS Code, por:
 
-- Open the *extensions* tab (_View_ → _Extensions)_ and search for Biome.
-- Open the _Quick Open Overlay_ (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd>P</kbd> or _Go -> Go to File_), enter `ext install biomejs.biome`, and hit enter.
+- Abrir a aba *extensões* (_Exibir_ → _Extensões)_ e procurar por Biome.
```
[the remark this answers is not in the document]
Comment (victor-teles) [reply website/src/content/docs/pt-br/reference/vscode.mdx:17]: Usually when we instruct the user we use "Abra" and not "Abrir". (imperative form)
"Abrir" is not that incorrect but "Abra" make more sense in that case

**Review by victor-teles — CHANGES_REQUESTED** [review]
@ImBIOS Thank you for the contribution!

I left some suggestions :)

File: website/src/content/docs/pt-br/reference/vscode.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (ImBIOS) [reply website/src/content/docs/pt-br/reference/vscode.mdx:17]: Obrigado por isso, eu agradeço.

File: website/src/content/docs/pt-br/reference/vscode.mdx [hunk 2]
```diff
@@ -1,76 +1,78 @@
...
-## Installation
+## Instalação
 
-You can install the code extension by heading to the extension's [Visual Studio Code Market Place page](https://marketplace.visualstudio.com/items?itemName=biomejs.biome) or from within VS Code by either:
+Você pode instalar a extensão do código acessando a [página do Visual Studio Code Market Place da extensão](https://marketplace.visualstudio.com/items?itemName=biomejs.biome) ou a partir do VS Code:
 
-- Open the *extensions* tab (_View_ → _Extensions)_ and search for Biome.
-- Open the _Quick Open Overlay_ (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd>P</kbd> or _Go -> Go to File_), enter `ext install biomejs.biome`, and hit enter.
+- Abra a aba *extensões* (_Exibir_ → _Extensões)_ e procure por Biome.
+- Abra a _Sobreposição de Abertura Rápida_ (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd>P</kbd> ou _Ir -> Ir para Arquivo_), digite `ext install biomejs.biome` e pressione enter.
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/reference/vscode.mdx:18]:
```suggestion
- Abra a _Sobreposição de Abertura Rápida_ (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd>P</kbd> ou _Acessar -> Ir para Arquivo_), digite `ext install biomejs.biome` e pressione enter.
```

Just a small suggestion, I noticed that in the Portuguese version of vs code, the menu is called "Acessar" and not "Ir"

File: website/src/content/docs/pt-br/reference/vscode.mdx [hunk 3]
```diff
@@ -1,76 +1,78 @@
...
+### Formatador Padrão
 
-Configure Biome as the default formatter for supported files to ensure that VS Code uses Biome over other formatters that you may have installed. You can do so by opening a JavaScript or TypeScript and then:
+Configure o Biome como o formatador padrão para arquivos suportados para garantir que o VS Code use o Biome em vez de outros formatadores que você possa ter instalado. Você pode fazer isso abrindo um JavaScript ou TypeScript e depois:
 
-- Open the Command Palette (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd title="Shift">⇧</kbd>+<kbd>P</kbd> or View → Command Palette)
-- Select _Format Document With…_
-- Select _Configure Default Formatter…_
-- Select Biome
+- Abra a Paleta de Comandos (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd title="Shift">⇧</kbd>+<kbd>P</kbd> ou Exibir → Paleta de Comandos)
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/reference/vscode.mdx:26]:
```suggestion
- Abra a Paleta de Comandos (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd title="Shift">⇧</kbd>+<kbd>P</kbd> ou Ver → Paleta de Comandos)
```
 I noticed that in the Portuguese version of vs code, the menu is called "Ver" and not "Exibir"

**Review by victor-teles — APPROVED** [review]
@ImBIOS Looks good to me.
I left some really small suggestions to make it more close to VS code menu names

File: website/src/content/docs/pt-br/reference/vscode.mdx (hunk 2 above)
Comment (ImBIOS) [reply website/src/content/docs/pt-br/reference/vscode.mdx:18]: nice catch, I just translated according to my experience.

File: website/src/content/docs/pt-br/reference/vscode.mdx (hunk 3 above)
Comment (ImBIOS) [reply website/src/content/docs/pt-br/reference/vscode.mdx:26]: nice catch, I just translated according to my experience.

**Pull request merged**